app/controllers/api/v1/registrations_controller.rb

```diff
@@ -263,11 +263,21 @@ def validate_payment_ticket_request
   end
 
   def payment_ticket
+    # TODO: GB Remove this default after deployment, once we're sure all existing competitions' requests
```

Comment:
Should we remove this now?

Reply:
Done

Reply:
Not done - see below comment chain

Reply:
I don't follow... Please link me to the exact comment that justifies why you cannot remove this default value computation here.